REQS.md

```diff
@@ -25,7 +24,6 @@ are defined, and so on). There are many, many viable ways to do it.
 
 Resources that are "modified" by one or more of the CSpace modules are:
 
-- EFS: access points are created
```

Comment:
Minor: worth reinstating this? Alternatively, just delete this file. It's not likely anyone else with will be using this.